docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -112,6 +113,42 @@ If UserInfo is the source of the roles then set `quarkus.oidc.authentication.use
 
 Additionally, a custom `SecurityIdentityAugmentor` can also be used to add the roles as documented in xref:security-customization.adoc#security-identity-customization[Security Identity Customization].
 
+[[token-scopes-and-security-identity-permissions]]
+=== Token Scopes And SecurityIdentity Permissions
```

Comment:
```suggestion
=== Token scopes and SecurityIdentity permissions
```

Reply:
I think this might be closer to how the Doc team prefers it

Reply:
titles right before and after are capitalized, it will look strange, but I'll change it. Thank you

Reply:
btw I didn't figure that's what docs warning meant, otherwise I'd change it as well automatically. now that I read your suggestion, it's completely clear.

Reply:
@michalvavrik I'm fine with either form but I recall, Michelle was making similar suggestions, @michelle-purcell, can you please have a quick look at the doc updates ?

Reply:
I happen to know @michelle-purcell has busy day, I made suggested changes, let's see what docs rule says about that.